sdk-autoconfigure/src/main/kotlin/io/datalbry/connector/autoconfigure/bootstrap/AlxndriaSchemaBootstrapAutoConfiguration.kt

```diff
@@ -0,0 +1,23 @@
+package io.datalbry.connector.autoconfigure.bootstrap
+
+import io.datalbry.alxndria.client.api.PlatformClient
+import io.datalbry.connector.sdk.ConnectorProperties
+import io.datalbry.connector.sdk.bootstrap.AlxndriaSchemaBootstrap
+import io.datalbry.connector.sdk.schema.SchemaProvider
+import org.springframework.boot.autoconfigure.condition.ConditionalOnClass
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
+@Configuration
+@ConditionalOnClass(PlatformClient::class, SchemaProvider::class, ConnectorProperties::class)
+open class AlxndriaSchemaBootstrapAutoConfiguration {
```

Comment:
Nope. Has nothing to do with Alxndria. It's a Precise Schema not a Alxndria schema

Reply:
I have renamed it accordingly. Thank you for you feedback!